src/main/java/org/jabref/gui/preftabs/AppearancePrefsTab.java

```diff
@@ -54,12 +40,12 @@ public AppearancePrefsTab(DialogService dialogService, JabRefPreferences prefs)
         fontSizeContainer.disableProperty().bind(overrideFonts.selectedProperty().not());
         fontTweaksLAF = new CheckBox(Localization.lang("Tweak font rendering for entry editor on Linux"));
 
-        VBox container = new VBox();
         container.getChildren().addAll(overrideFonts, fontSizeContainer, fontTweaksLAF);
 
-        JFXPanel panel = CustomJFXPanel.wrap(new Scene(container));
-        panel.setBorder(BorderFactory.createEmptyBorder(5, 5, 5, 5));
-        add(panel, BorderLayout.CENTER);
+    }
+
+    public VBox getContainer() {
```

Comment:
Better set the return type to Node, it's always better to return the Interface Type (https://www.quora.com/What-is-the-use-of-having-an-interface-as-a-return-type-in-Java)